packages/back-end/src/api/features/listFeatures.ts

```diff
@@ -19,7 +19,7 @@ export const listFeatures = createApiRequestHandler(listFeaturesValidator)(
   async (req): Promise<ListFeaturesResponse> => {
     const features = await getAllFeatures(req.context, {
       project: req.query.projectId,
-      includeArchived: true,
+      includeArchived: req.query.includeArchived || true,
```

Comment:
I don't think this will work since if you set it to false, it will do `false || true` and will still evaluate as true.